src/reference/02-DetailTopics/01-Using-sbt/07-sbt-server.md

```diff
@@ -104,13 +104,19 @@ By default, sbt server will be running when a sbt shell session is active. When
 }
 ```
 
-This gives us three pieces of information:
+or 
 
-1. That the server is (likely) running.
-2. That the server is running on port 5010.
-3. The location of the token file.
+```json
+{
+  "uri":"local:sbt-server-0845deda85cb41abdb9f",
+}
+```
```

Comment:
Assuming more people are on Mac and Linux, it might be better to use domain socket as the first example, and move TCP to later.

```json
{"uri":"local:///Users/someone/.sbt/1.0/server/0845deda85cb41abdb9f/sock"}
```

Reply:
I split the examples in three cases, leading with domain sockets, then named pipes, then TCP.